src/data/directory/lgu.json

```
@@ -10048,11 +10049,41 @@
           }
         ],
         "municipalities": [
+          {
+            "municipality": "Alcantara",
+            "zip_code": "6030",
```

References: 
- https://www.philippineszipcode.com/6033/
- https://discoveringcebu.com/cebu-maps/cebu-postal-code/
```suggestion
            "zip_code": "6033",
```